quickstart/gradle.properties

```diff
@@ -0,0 +1 @@
+android.enableDexingArtifactTransform.desugaring=false
```

Comment:
I think we can just set this at the top level `gradle.properties`

Reply:
Spent some time trying to get this to work, but with no luck. Here's the error that I am seeing when adding that line to the top level gradle.properties file.

```
FAILURE: Build failed with an exception.

* Where:

FAILURE: Build failed with an exception.

* Where:
Script '/Users/jaltonqualls/workspace/twilio/video-quickstart-android/video-quickstart-android-fork/gradle.properties' line: 1

* What went wrong:
A problem occurred evaluating script.
> Could not get unknown property 'android' for project ':exampleAdvancedCameraCapturer' of type org.gradle.api.Project.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

CONFIGURE FAILED in 0s
Could not get unknown property 'android' for project ':exampleAdvancedCameraCapturer' of type org.gradle.api.Project.
```

Reply:
Reference #439 for how to resolve these issues.